effects/steal_effect.py

```diff
@@ -1,14 +1,13 @@
 from game import Game
 from effects.effect import Effect
 from citizens.citizen import Citizen
-from citizens.spy import Spy
 import user_interaction
 import utils
 
 
 class StealEffect(Effect):
 
-    def __init__(self, name: str, game: Game, creator: Citizen) -> None:
+    def __init__(self, name: str, game: Game, creator: Citizen):
```

Comment:
You should use -> None type hint for every method whitch doesn't return any value.
__init__ is not an exception

Reply:
Done